overviews/quasiquotes/definition-details.md

```diff
@@ -126,15 +126,15 @@ Each val and var consistents of four components: modifiers, name, type tree and
     tpt: universe.Tree = <type ?>
     rhs: universe.Tree = 2
 
-If type of the val isn't explicitly specified by the user an [empty type](/overviews/quasiquotes/type-details.html#empty-type) is used as tpt.
+If the type of the `val` isn't explicitly specified by the user an [empty type](/overviews/quasiquotes/type-details.html#empty-type) is used as `tpt`.
 
-Vals and vars are disjoint (they don't match one another):
+`Val`s and `var`s are disjoint (they don't match one another):
```

Comment:
If it's in backquotes, it's literal and shouldn't have an initial capital letter, even at the start of a sentence.